assets/js/modules/analytics/components/settings/SettingsForm.js

```diff
@@ -36,7 +38,10 @@ export default function SettingsForm() {
 	return (
 		<div className="googlesitekit-analytics-settings-fields">
 			<StoreErrorNotice moduleSlug="analytics" storeName={ STORE_NAME } />
+			<ExistingGTMPropertyError />
```

Comment:
Instead of putting this here, let's add it similarly to how the `ExistingTagError` is placed, within the main `SettingsEdit` component. It should fully replace all the rest of the editing UI and prevent changing settings.

Reply:
Updated.